### PR TITLE
PR CL-129-Add-Cors-To-CRMMicroservice

### PR DESCRIPTION
**Pull request by OleksandraKondratenko**

### CODE REVIEW

File: CRMLite.CRMAPI/Controllers/LeadController.cs [hunk 1]
```diff
@@ -10,7 +10,7 @@ namespace CRMLite.CRMAPI.Controllers
 {
     [ApiController]
     [Route("api/[controller]")]
-    [Authorize(Policy = "PermissionForAdminAndUserRoles")]
+    //[Authorize(Policy = "PermissionForAdminAndUserRoles")]
```
Comment (Aryewar) [inline CRMLite.CRMAPI/Controllers/LeadController.cs]: Delete commit

File: CRMLite.CRMDAL/Repositories/LeadRepository.cs [hunk 2]
```diff
@@ -50,7 +50,15 @@ public async Task RegistrationLeadAsync(Lead lead)
...
             {
-                await _leadRepository.RegistrationLeadAsync(lead);
+                try
+                {
+                    await _leadRepository.RegistrationLeadAsync(lead);
+                }
+                catch(Exception e)
+                {
+                    throw e;
+                }
```
Comment (Aryewar) [inline CRMLite.CRMDAL/Repositories/LeadRepository.cs]: Remove try/catch

File: CRMLite.CRMAPI/Controllers/LeadController.cs (hunk 1 above)
Comment (MykhailoKnysh) [inline CRMLite.CRMAPI/Controllers/LeadController.cs]: Remove comments

File: CRMLite.CRMAPI/Startup.cs [hunk 3]
```diff
@@ -38,6 +39,18 @@ public void ConfigureServices(IServiceCollection services)
...
+                options.AddPolicy(name: MyAllowSpecificOrigins,
+                                  builder =>
+                                  {
+                                      builder.WithOrigins("http://localhost:3000", "http://localhost:1234")
+                                     .AllowAnyHeader()
+                                     .AllowAnyMethod()
+                                     .AllowAnyOrigin();
+                                  });
+            });
+
```
Comment (MykhailoKnysh) [inline CRMLite.CRMAPI/Startup.cs]: Try Ctrl K+D

**Review by Aryewar — APPROVED** [review]

**Review by MykhailoKnysh — APPROVED** [review]

**Pull request merged**